src/db.h

```diff
@@ -38,6 +38,7 @@ struct db_api_arg {
 	scmp_datum_t datum;
 
 	bool valid;
+	bool is_32bit;
```

Comment:
I think I'd prefer to drop the `is_32bit` flag and just preserve the `SCMP_CMP_XXX` flags in the `db_api_arg::op` field; there aren't that many places that check the comparison op so masking them shouldn't be too terrible.  If necessary we can always wrap it in a preprocessor macro, e.g.
```
#define CMP_OP(x) (x & __SCMP_CMP_OPMASK)
#define CMP_FLAGS(x) (x & __SCMP_CMP_FLAGMASK)
```